src/skins/vector/css/matrix-react-sdk/views/dialogs/_CreateGroupDialog.scss

```diff
@@ -33,3 +33,30 @@ limitations under the License.
     background-color: $primary-bg-color;
 }
 
+.mx_CreateGroupDialog_input.has_prefix.has_suffix {
```

Comment:
we *have* to namespace CSS classes correctly, otherwise this will collide with any other random CSS dependency (e.g. markdown, syntax highlighter, etc) which might choose to define a generic class like `has_prefix`.  In this instance, I assume it should be something like `mx_CreateGroupDialog_input_hasPrefix`.

....however, it's very very unclear and confusing as to what the difference is between `mx_CreateGroupDialog_input_prefix` and `mx_CreateGroupDialog_input.has_prefix` here, and so i suspect you need a better name anyway.

Reply:
ah, I see what you're saying

let's go with 
`.mx_CreateGroupDialog_prefix` and `.mx_CreateGroupDialog_input_hasPrefix`

Reply:
also, i would avoid the `.foo.bar.baz` selector format - we don't use it in many other places, and instead do the (different) `.foo .bar, .foo .baz` selector to say "select classes with name bar or baz which are within a parent class called foo".   (Unless there's a performance improvement I've forgotten about explicitly having `.foo.bar.baz` being "select classes which have names foo, bar & baz"?  I may also be a bit historically traumatised by `.foo.bar.baz` not working reliably on IE4 ;P)